C-Plus-Plus/math/Find_Even_Or_Odd_without_arithmetic_operators.cpp

```diff
@@ -0,0 +1,64 @@
+/*
+Author - Nikitha Reddy [Github ID - Nikitha2309]
```

Comment:
Remove this line

Reply:
Done , Author Name removed from the code